tests/ui/test_creation_and_deletion_of_sc_and_rbdpool.py

```diff
@@ -21,6 +22,7 @@
 log = logging.getLogger(__name__)
 
 
+@black_squad
```

Comment:
The test was created and is maintained by Green Squad.
@shivamdurgbuns please confirm or reject.

Reply:
Updated

Reply:
thanks